src/xenia/kernel/kernel_state.h

```diff
@@ -388,6 +388,8 @@ class KernelState {
 
  public:
   uint32_t dash_context_ = 0;
+  uint64_t dash_app_info_unk1 = 0;
+  uint32_t dash_app_info_unk2 = 0;
```

Comment:
Instead of declaring 2 values just use X_DASH_APP_INFO. ``X_DASH_APP_INFO  dash_app_info_ = {}``